lib/vc.js

```diff
@@ -120,14 +117,19 @@ async function issue(options = {}) {
     throw new TypeError('"suite.verificationMethod" property is required.');
   }
 
-  // run common credential checks
-  const {credential} = options;
   if(!credential) {
     throw new TypeError('"credential" parameter is required for issuing.');
   }
+
+  // Set the issuance date to now(), if missing
+  const now = (new Date()).toJSON();
+  credential.issuanceDate = credential.issuanceDate ||
+    `${now.substr(0, now.length - 5)}Z`;
```

Comment:
Could refactor this to only run the new date code if `credential.issuanceDate` isn't set.

Reply:
Done.